tlsproxy.go

```diff
@@ -1,109 +1,24 @@
 // tlsproxy provides a TLS proxy kind of like stunnel
-package main
+package proxy
```

Comment:
I would call this tlsproxy so the package name matches the path.

Reply:
done.